frontend/simple/components/DashboardSection.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <section class="gi-section">
+    <h3 class="title"><i18n>{{title}}</i18n></h3>
```

Comment:
Hmm. This is a problem. Strings files for translation are generated from the source by searching it for text that's wrapped in `<i18n>` tags. This is done while the app is not running. So here the string-to-be-translated would be `{{title}}`, clearly not what's intended.

A potential fix for this is to remove the `<i18n>` tag here (just have it be `{{title}}`), and in the parent component pass in the prop like so:

```js
<dashboard-section :title="L('Support History')">
```

You will need to import the `L` function in whatever files you do this in.

Reply:
Oh thanks for letting me know, I didn't understand how that worked behind.

One thing I've noticed too is that `<i18n>` returns a `<span>` with the content inside, can we remove the <span>? With it we have an extra node for every text on the DOM for no purpose.
```diff
<template>
-    <span><slot></slot></span>
+    <slot></slot>
</template>
```

Reply:
@taoeffect Instead of importing `L()` in every file that needs a translation to `dashboard-section` (every file), what about make it on the component itself. If the title is not translated, use `title` prop, if it comes already translated, use `titleL`

```
// DashboardSection.vue
<h3 class="title">{{sectionTitle}}</h3>

// ...

  props: {
    title: String,
    titleL: String
  },
  computed: {
    sectionTitle () {
      return this.titleL || L(this.title)
    }
  }
```

```vue
// Text not translated
<dashboard-section title="Support History">
```

or

```vue
// Text translated
<dashboard-section :titleL="sectionTitle">

computed: {
    sectionTitle () {
      return `${this.period} ${L('overview')}` // July Overview translated
    }
}
```
Does this works?

Reply:
> One thing I've noticed too is that <i18n> returns a <span> with the content inside, can we remove the ?

I don't know if the `<span>`s can be removed. I'm guessing I put them there because without them vue would complain, but I could be wrong, maybe it works now. You're welcome to try.

>  If the title is not translated, use title prop, if it comes already translated, use titleL

We will have a much cleaner way of doing it than that that. Right now don't worry too much about it.